activerecord/lib/active_record/database_configurations/database_config.rb

```diff
@@ -21,6 +21,13 @@ def migrations_paths
         raise NotImplementedError
       end
 
+      # The adapter for a database configuration. If the
+      # `adapter` key is present in the config, `adapter`
+      # will return its value.
```

Comment:
Seems needlessly verbose for such a simple implementation? Since this class is `nodoc` and the logic ain't complex, let's just skip the documentation.